WordPress/gutenberg
# [RNMobile] Final Page Templates with localization

Waiting for https://github.com/WordPress/gutenberg/pull/19106, rebase to `master` before it's ready for final review.

## Description


This seems like the most sensible approach to template i18n for now. It would add some friction to creating and maintaining the templates, but we could create some tooling around that so we can create templates in Gutenberg, then export a JSON. It needs some comments for translators to clarify that it’s example content, but other than that this might be it

For https://github.com/wordpress-mobile/gutenberg-mobile/issues/1473

## How has this been tested?

- Open the main app with **metro running**
- Go to pages
- Create a new page
- Select a template
- **Expect**: to see the modal with the preview
- Tap on **Apply**
- **Expect**: the content of the preview to be in the editor and the modal should close

## Screenshots 

About|Contact
-|-
<img src="https://user-images.githubusercontent.com/4885740/74425299-c1dd9d80-4e53-11ea-93a5-f33d1d4741a2.png" width="250" />|<img src="https://user-images.githubusercontent.com/4885740/74425364-dcb01200-4e53-11ea-838c-03afd1e45229.png" width="250" />

Portfolio|Services
-|-
<img src="https://user-images.githubusercontent.com/4885740/74425416-f0f40f00-4e53-11ea-90cd-56e34448185f.png" width="250" />|<img src="https://user-images.githubusercontent.com/4885740/74425420-f3566900-4e53-11ea-8ef1-86b1d3f5a03d.png" width="250" />

Team|Team
-|-
<img src="https://user-images.githubusercontent.com/4885740/74425488-0d904700-4e54-11ea-8b8c-fb94d1b6bab9.png" width="250" />|<img src="https://user-images.githubusercontent.com/4885740/74425584-3b758b80-4e54-11ea-98ff-2b5af2b3865d.png" width="250" />

## Types of changes
Adds final set of page templates for mobile with their localization

## Checklist:
